images/linux/scripts/installers/nodejs.sh

```diff
@@ -30,6 +30,11 @@ done
 echo "Creating the symlink for [now] command to vercel CLI"
 ln -s /usr/local/bin/vercel /usr/local/bin/now
 
+# fix global modules installation as regular user
+# related issue https://github.com/actions/virtual-environments/issues/3727
+sudo chmod 777 /usr/local/lib/node_modules -R
+sudo chmod 777 /usr/local/bin
```

Comment:
Is it a good idea to give such permissions to the whole `/usr/local/bin?`

Reply:
I guess yes, good idea!